src/actions/run-command.cpp

```diff
@@ -48,12 +48,19 @@ void RunCommand::executePrelude() {
   }
 }
 
-void RunCommand::executeAction(const Gesture& /*gesture*/) {
-  RunCommand::runCommand(this->command);
+void RunCommand::executeAction(const Gesture& gesture) {
+  RunCommand::runCommand(this->command, gesture);
 }
 
-void RunCommand::executeReverse(const Gesture& /*gesture*/) {
-  RunCommand::runCommand(this->decreaseCommand);
+void RunCommand::executeReverse(const Gesture& gesture) {
+  RunCommand::runCommand(this->decreaseCommand, gesture);
+}
+
+bool RunCommand::runCommand(const std::string& command,
+                            const Gesture& gesture) {
+  setenv("DEVICE_TYPE",
```

Comment:
What do you think about prefixing the variable with `TOUCHEGG_`? It'd help to avoid variables with the same name in the user script.